src/routes/(site)/about/officer-profile.svelte

```diff
@@ -22,21 +23,55 @@
     return 'null';
   }
 
-  // teamClasses maps team names to CSS class names. We call Object.entries on
-  // it to make it easier to iterate and search.
-  const teamClasses = Object.entries({
-    Algo: 'acm-purple',
-    Create: 'acm-pink',
-    Design: 'acm-pink',
-    Dev: 'acm-bluer',
-    AI: 'acm-emerald',
-    Marketing: 'acm-red',
-    'Special Events': 'acm-lemon',
-    nodebuds: 'brand-header',
-  });
+  /**
+   * teamColors maps team names to CSS color variables. We call Object.entries on
+   * it to make it easier to iterate and search.
+   */
+  const teamColors = {
```

Comment:
Nit: Rename `teamColors` to `COLORS` and move to `src/lib/public/board/data/colors.ts`. Export from `src/lib/public/board/data/index.ts`.